MarkBind/markbind
# Reimplement modal scrolling using Bootstrap's .modal-dialog-scrollable

**What is the purpose of this pull request?**


Resolves #2362

Reimplement PR #2322



**Overview of changes:**

We can directly use Bootstrap's provided class `.modal-dialog-scrollable` (see https://getbootstrap.com/docs/5.1/components/modal/#scrolling-long-content), rather than the cumbersome way of adding custom styles (`max-height` and `overflow-auto`) previously done in PR #2322.

- Replace custom styles added previously in PR #2322 by adding `.modal-dialog-scrollable` to modal-content.

**Anything you'd like to highlight/discuss:**


**Testing instructions:**
Run `npm test` to ensure snapshot tests are updated, and

1. cd into `packages/cli/test/functional/test_site` and run `markbind serve -d`
1. navigate to `test_site/testModals.html` in browser and test out _Trigger for modal that should overflow_



**Proposed commit message: (wrap lines at 72 characters)**
Reimplement scrolling using .modal-dialog-scrollable


---

**Checklist:** :ballot_box_with_check:
